src/dpp/dispatcher.cpp

```diff
@@ -43,6 +43,13 @@ void interaction_create_t::reply(interaction_response_type t, const std::string
 	this->reply(t, dpp::message(this->command.channel_id, mt, mt_application_command));
 }
 
+void interaction_create_t::get_original_response(command_completion_event_t callback) const
+{
+	from->creator->post_rest(API_PATH "/webhooks", std::to_string(command.application_id), command.token + "/messages/@original", m_get, "", [callback](json &j, const http_request_completion_t& http) {
+		callback(confirmation_callback_t("message", message().fill_from_json(&j), http));
```

Comment:
should probably check callback is valid before calling it, in case someone passes in an uninitialised `std::function`

Reply:
I didn't want branching there because it mostly it's a fixed function that is passed and would error out, saving us from branching, but I get the point you make. If you want, I can add it.